File: Lib/NexssLog.js
const nxsInfo = (...info) => {
  console.error(`NEXSS/info:`, ...info);
};

const nxsOk = (...ok) => {
  console.error(`NEXSS/ok:`, ...ok);
};

const nxsError = (...error) => {
  console.error(`NEXSS/error:`, ...error);
};

const nxsWarn = (...warn) => {
  console.error(`NEXSS/warn:`, ...warn);
};

const nxsTrace = (...trace) => {
  console.error(`NEXSS/trace:`, ...trace);
};

const nxsDebug = (...debug) => {
  console.error(`NEXSS/debug:`, ...debug);
};

module.exports = { nxsInfo, nxsOk, nxsError, nxsWarn, nxsTrace, nxsDebug };

File: Lib/NexssLog.py
# Python Log for Nexss Programmer
import sys

def nxsInfo(info):
    sys.stderr.write(f'NEXSS/info: {info}')

def nxsOk(ok):
    sys.stderr.write("NEXSS/ok: %s" % ok)

def nxsError(error):
    sys.stderr.write("NEXSS/error: %s" % error)

def nxsWarn(warn):
    sys.stderr.write("NEXSS/warn: %s" % warn) 

def nxsDebug(debug):
    sys.stderr.write("NEXSS/debug: %s" % debug) 